tests/test_fp4_quantize.py

```diff
@@ -312,7 +311,7 @@ def test_e2m1_dequantization(
 
     # Test with default common settings
     is_sf_swizzled_layout = True
-    block_size = 32 if sf_use_ue8m0 else 16
+    block_size = 16  # 32 if sf_use_ue8m0 else 16
```

Comment:
what motivate this change?

Reply:
This for mxfp4. The trtllm upstream only supports 16.